### ISSUE TITLE
Boleto Registrado Caixa

### ISSUE THREAD

**Issue by celsinhogarcia** [BoletoNet/boletonet#328]
Pessoal, tudo bom?

Já conseguimos homologar o boleto registrado da Caixa em nossa solução? A carteira que pergunto é a CR.

Muito obrigado, abraços.

**Comment by rodolfoghi** [comment]
Sim, temos clientes que utilizam a carteira 01 (Cobrança Registrada) que no boleto é impresso RG, conforme o manual que pode ser obtido em [https://www.caixa.gov.br/Downloads/cobranca-caixa/Manual_Leiaute_CNAB400_SIGCB.pdf](https://www.caixa.gov.br/Downloads/cobranca-caixa/Manual_Leiaute_CNAB400_SIGCB.pdf)
